### ISSUE TITLE
simdkalman filter processor issues

### ISSUE THREAD

**Issue by tomjrtsmith** [nateemma/strategies#3]
Hey Phill, I've been trying to run several of your KalmanSIMD strats and I'm seeing (I think...) the simdkalman package consume an entire processor thread which won't allow the strat to run. Wondered if you'd experienced the same or knew a workaround.

**Comment by nateemma** [comment]
It worked fine for me, but I run on a Mac, so it could be different. Also, I haven't touched those strategies in many months - I found that the predictions didn't work well for simdkalman, plus they are very slow (as you mention). There are also the 'regular' Kalman filter Strats that use pykalman

**Comment by tomjrtsmith** [comment]
I ran it on a vps instance and it wouldn't multithread, I think because of the SIMD filter. I've found the DWT strats work best so far, still working my way through them all.

**Comment by tomjrtsmith** [comment]
sklearn is throwing out this error when I got Anomaly running. I'm also noticing some memory bleed, I think from tensorflow (seems worse on the PCA/LSTM strats) but still noticeable using sklearn. I've had trouble getting finta to run so edited out VFI and made some edits in my fork around some of your custom settings and they seem to run ok. Accuracy is amazing. 

/home/ubuntu/freqtrade/.env/lib/python3.10/site-packages/sklearn/metrics/_classification.py:1334: UndefinedMetricWarning: Precision and F-score are ill-defined and being set to 0.0 in labels with no predicted samples. Use `zero_division` parameter to control this behavior.

**Comment by nateemma** [comment]
This happens when you get 0 buy/sell signals in the test data (because some
of the metrics divide by that).  It doesn’t affect logic, but it does
indicate that maybe the buy/sell triggers are too tight

Cheers

Phil

**Comment by nateemma** [comment]
BTW, not sure when you forked, but I stopped using VFI a while ago - it
sometimes returns infinite numbers, which messes up my normalisation code
for the PCA/NNBC/Predict/Anomaly classes

Also, accuracy from the PCA classes is 'too good', which implies there is
some lookahead bias somewhere. I think it might be in the stoploss/custom
sell code since the results are very dramatically affected by those hyper
parameters. Still trying to figure this one out...

Cheers,

Phil
